### PR TITLE
ENCD-6144 tracks batch

### PR DESCRIPTION
**Pull request by gabdank**

### CODE REVIEW

File: src/encoded/static/components/genome_browser.js [hunk 1]
```diff
@@ -597,6 +599,8 @@ const TrackLabel = ({ file, label, supplementalShortLabel, long }) => {
         ]).join(', ');
     }
 
+    const originBatchesList = file.origin_batches ? `batch${file.origin_batches.length > 1 ? 'es' : ''}: ${file.origin_batches.map((batch) => batch.split('/')[2]).join(', ')}` : '';
```
Comment (zoldello) [inline src/encoded/static/components/genome_browser.js:602]: This code is fine. Some feel though it makes code unnecessarily to have nested ternary operators.

**Review by zoldello — DISMISSED** [review]
Code looks good.

File: src/encoded/static/components/genome_browser.js (hunk 1 above)
Comment (gabdank) [reply src/encoded/static/components/genome_browser.js:602]: thanks.
I am not an expert in JS, but a one-liner looked better than 3 lines.

**Review by zoldello — APPROVED** [review]
Code looks good.

File: src/encoded/static/components/genome_browser.js (hunk 1 above)
Comment (hitz) [reply src/encoded/static/components/genome_browser.js:602]: For future considerations I think 3 lines is more comprehensible unless you break it up with <returns>

**Pull request merged**